UI/BindingSources/XAML/Binding/MainPage.xaml

```diff
@@ -0,0 +1,39 @@
+﻿<Page x:Class="Binding.MainPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Binding"
+	  xmlns:um="using:Uno.Material"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  x:Name="page"
+	  Background="{ThemeResource BackgroundBrush}">
+
+	<Page.DataContext>
+		<local:MainViewModel />
+	</Page.DataContext>
+
+	<StackPanel HorizontalAlignment="Center"
+				VerticalAlignment="Center"
+				utu:SafeArea.Insets="VisibleBounds">
+		<ListView ItemsSource="{Binding Items}">
+			<ListView.ItemTemplate>
+				<DataTemplate>
+					<StackPanel Orientation="Horizontal" Spacing="8">
+						<TextBlock VerticalAlignment="Center" Text="{Binding}" />
+						<Button Command="{utu:ItemsControlBinding Path=DataContext.RemoveItemCommand}"
+								CommandParameter="{Binding}"
+								Content="Delete" />
+						<StackPanel.ContextFlyout>
+							<MenuFlyout>
+								<MenuFlyout.Items>
+									<MenuFlyoutItem Command="{Binding DataContext.RemoveItemCommand, ElementName=page}"
```

Comment:
Using `{utu:ItemsControlBinding Path=DataContext.RemoveItemCommand}` here won't work. I guess because it's inside a MenuFlyout the `DataContext` gets messed up.

![image](https://github.com/unoplatform/Uno.Samples/assets/114886335/6a479004-c2ae-44c8-be13-38ad208a2e13)

@kazo0 @dansiegel

Reply:
See my answer here: https://github.com/unoplatform/Uno.Samples/pull/681/files#r1597206927